gokart/testing/check_if_run_with_empty_data_frame.py

```diff
@@ -81,7 +81,8 @@ def _test_run_with_empty_data_frame(cmdline_args: List[str], test_run_params: te
 
 def try_to_run_test_for_empty_data_frame(cmdline_args: List[str]):
     with CmdlineParser.global_instance(cmdline_args):
-        test_run_params = test_run()
+        # NOTE: parameters passed by command line cannot be linted by mypy.
+        test_run_params = test_run()  # type: ignore
```

Comment:
mypy-plugin cannot pass this case even though the parameters passed from `cmdlin_args`